src/rantanplan/core.py

```diff
@@ -38,9 +54,1266 @@
 LIAISON_FIRST_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚy")
 LIAISON_SECOND_PART = set("aeiouáéíóúAEIOUÁÉÍÓÚhy")
 STRESSED_UNACCENTED_MONOSYLLABLES = {"yo", "vio", "dio", "fe", "sol", "ti", "un"}
-UNSTRESSED_UNACCENTED_MONOSYLLABLES = {"me", "nos", "te", "os", "lo", "la", "los", "las", "le",
-                                       "les", "se", "tan", "el", "mas", "te", "si", "tu", "de",
-                                       "mi", "si", "tu", "que", "de", "su"}
+UNSTRESSED_UNACCENTED_MONOSYLLABLES = {'de', 'el', 'la', 'las', 'le', 'les', 'lo', 'los',
+                                       'mas', 'me', 'mi', 'nos', 'os', 'que', 'se', 'si',
+                                       'su', 'tan', 'te', 'tu'}
+
+
+"""
+Regular expressions and rules for syllabification exceptions
+
+"""
+
+
+"""
+Fonetic exceptions dict for potential diaeresis
+
+paranoia': (['pa', 'ra', 'noia'],
+         [
+         (['pa', 'ra', 'no', 'ia'], (2, 3)), # Metaplasm positions
+         (['pa', 'ra', 'no', 'i', 'a'], (2, 3, 4))  # Another, fake, metaplasm positions
+         ]),
+"""
+
+ALTERNATE_PHONETIC_SYLLABIFICATION = {'arcaizar': (['ar', 'cai', 'zar'], [(['ar', 'ca', 'i', 'zar'], (1, 2))]),
+                                      'arcaizando': (['ar', 'cai', 'zan', 'do'],
+                                                     [(['ar', 'ca', 'i', 'zan', 'do'], (1, 2))]),
```

Comment:
Let's change all indentations so it doesn't go too far:
```python
ALTERNATE_PHONETIC_SYLLABIFICATION = {
    'arcaizar': (['ar', 'cai', 'zar'], [(['ar', 'ca', 'i', 'zar'], (1, 2))]),
    'arcaizando': (['ar', 'cai', 'zan', 'do'],
                   [(['ar', 'ca', 'i', 'zan', 'do'], (1, 2))]),
...
```

Moreover, I think `ALTERNATIVE_SYLLABIFICATIONS` might be more correct

Reply:
Done